Pages/slider.html

```diff
@@ -0,0 +1,166 @@
+                    <a class="sidebar-item" href="#">Understand Orca</a>
+                    <a class="sidebar-item" href="#">Colors</a>
+                    <a class="sidebar-item" href="#">Typography</a>
+                </li>
+            </ul>
+            <h3 class="sidebar-heading"><a href="#">Layout</a></h3>
+        <ul>
+          <li>
+            <a class="sidebar-item" href="/Pages/grid.html">Grid </a>
+          </li>
+        </ul>
+            <h3 class="sidebar-heading"><a href="#">Components</a></h3>
+            <ul>
+                <li>
+                    <a class="sidebar-item" href="/Pages/avatar.html">Avatar
+                    </a>
+                    <a class="sidebar-item" href="/Pages/badge.html">Badge</a>
+                    <a class="sidebar-item" href="/Pages/alert.html">Alert</a>
+                    <a class="sidebar-item" href="/Pages/button.html">Button</a>
+                    <a class="sidebar-item" href="/Pages/card.html">Card</a>
+                    <a class="sidebar-item" href="/Pages/input.html">Input</a>
+                    <a class="sidebar-item" href="/Pages/image.html">Image</a>
+                    <a class="sidebar-item" href="/Pages/modal.html">Modal</a>
+                    <a class="sidebar-item" href="/Pages/navigation.html">Navigation</a>
+                    <a class="sidebar-item list-item-active" href="/Pages/slider.html">Slider</a>
+                </li>
+            </ul>
+        </div>
+        <div class="main-content">
+            <h1 class="my-2">Slider</h1>
+            <hr>
+            <p>Slider is an input type which is useful in various places where user is required to input a number or a value but in a graphical format. Input slider makes it easy and interactive for user to input a value.
+            </p>
+            <hr>
+            <h3 class="my-2">How to use</h3>
+            <p>Copy paste the below code snip in your body tag and adjust according to your need. </p>
+            <h3 class="my-2">Examples</h3>
+            <div class="example-box my-4">
+                <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=seti&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Clabel%2520for%253D%2522input%2522%253EPrice%2520range%253C%252Flabel%253E%250A%253Cinput%2520id%253D%2522input-slider%2522%2520type%253D%2522range%2522%2520min%253D%25220%2522%2520max%253D%252210000%2522%253E"
+  style="width: 511px; height: 95px; border:0; transform: scale(1); overflow:hidden;"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+            </div>
+            <div class="example-box flex-col my-4">
+                <label for="input">Price range</label>
+                <input id="input-slider" type="range" min="0" max="10000">
+            </div>
+            <hr>
```

Comment:
just a suggestion you can also use a border-bottom instead of hr

Reply:
Its kind of the theme of the website but thanks for the suggestion